src/components/grapher/Grapher.vue

```diff
@@ -188,7 +207,8 @@ export default Vue.extend({
   left: 0;
 }
 
-.grapher--line-container {
+.grapher--line-container{
+  fill: $soybean;
```

Comment:
We should be careful about adding `fill` to all the lines. `fill` should only be for the field rectangle.